InvenTree/common/api.py

```diff
@@ -102,6 +103,36 @@ def _get_webhook(self, endpoint, request, *args, **kwargs):
             raise NotFound()
 
 
+class CurrencyExchangeView(APIView):
+    """API endpoint for displaying currency information
+
+    TODO: Add a POST hook to refresh / update the currency exchange data
```

Comment:
Question: Is this TODO for this PR or later? Should we maybe create a FR so we do not foget?

Reply:
Yes, I'll add a FR for this. Currently we do this with a web-based view (which should be removed)